teamwork/templates/base.html

```diff
@@ -1,23 +1,78 @@
-{% load static %}
+  {% load static %}
 
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <title>
+      {% block title %}
+          MyProjects
+      {% endblock %}
+      </title>
+      <meta name="description" content="Website for managing multiple projects in one place.">
+      <!-- Bootstrap CSS -->
+      <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
 
-    <title>
-    {% block title %}
-        MyProjects
-    {% endblock %}
-    </title>
-    <meta name="description" content="Website for managing multiple projects in one place.">
-    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap-theme.min.css' %}">
-    <link rel="stylesheet" href="{% static 'bootstrap/css/bootstrap.min.css' %}">
-    <link rel="stylesheet" href="{% static 'css/base.css' %}">
-  </head>
-  <body>
-    {% block content %}
-    {% endblock %}
-  </body>
-</html>
+      <!-- Custom CSS -->
+      <link rel="stylesheet" href="{% static 'css/main.css' %}">
+    </head>
+    <body>
+      <!-- Navigation bar -->
+      <nav class="navbar navbar-inverse navbar-static-top" role="navigation">

```

Comment:
The indentation between <nav> and <div class="container"> is too big and makes reading this file slightly more difficult.